### PR TITLE
Merge fixes for PR96

### PR DESCRIPTION
**Pull request by BrianFarnhill**

### CODE REVIEW

**Comment by BrianFarnhill** [comment]
@camiloborges - just finishing my testing on this now, once you merge this in it will automatically hit your main PR and then we can merge that and close it. Have a look over what i've done and let me know what you think.

File: Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,244 @@
+[CmdletBinding()]
+param(
+    [string] $SharePointCmdletModule = (Join-Path "C:\users\brian\Documents\GitHubVisualStudio\xSharePoint\Tests\xSharePoint" "..\Stubs\SharePoint\15.0.4693.1000\Microsoft.SharePoint.PowerShell.psm1" -Resolve)
```
Comment (camiloborges) [inline Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1:3]: Brian check this out :d

Comment (BrianFarnhill) [inline Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1:3]: Thats my bad - have fixed it in the next PR :P

Comment (camiloborges) [inline Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1:3]: Hello. I also got that in my last commit. Perhaps you should sync before pushing ?

Comment (BrianFarnhill) [inline Tests/xSharePoint/xSharePoint.xSPWebApplication.Throttling.Tests.ps1:3]: Where? Look at PR #2 - I see all those references removed and replaced with $PSScriptRoot

**Pull request merged**